src/Uno.UI/UI/Xaml/Window.Skia.cs

```diff
@@ -0,0 +1,197 @@
+			}
+		}
+
+		private static bool _isMeasureQueued = false;
+		private static bool _isMeasuring = false;
+
+		internal static void InvalidateMeasure()
+		{
+			Current.InnerInvalidateMeasure();
+		}
+
+		internal void InnerInvalidateMeasure()
+		{
+			if (_window != null)
+			{
+				if (!_isMeasureQueued)
+				{
+					_isMeasureQueued = true;
+
+					CoreDispatcher.Main.RunAsync(CoreDispatcherPriority.Normal, () => {
+						try
+						{
+							_isMeasureQueued = false;
+
+							_isMeasuring = true;
+
+							var sw = Stopwatch.StartNew();
+							_window.Measure(Bounds.Size);
+							_window.Arrange(Bounds);
+							sw.Stop();
+
+							if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Debug))
+							{
+								this.Log().Debug($"DispatchInvalidateMeasure: {sw.Elapsed}");
+							}
+
+							InvalidateRender();
+						}
+						finally
+						{
+							_isMeasuring = false;
+						}
+					});
+				}
+
+				
+			}
+		}
+
+		internal void OnNativeSizeChanged(Size size)
+		{
+			var newBounds = new Rect(0, 0, size.Width, size.Height);
+
+			if (newBounds != Bounds)
+			{
+				if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Debug))
+				{
+					this.Log().Debug($"OnNativeSizeChanged: {size}");
+				}
+
+				Bounds = newBounds;
+
+				InvalidateMeasure();
+				RaiseSizeChanged(new WindowSizeChangedEventArgs(size));
+			}
+		}
+
+        public Compositor Compositor { get; }
+
+        partial void InternalActivate()
+		{
+			// WebAssemblyRuntime.InvokeJS("Uno.UI.WindowManager.current.activate();");
+		}
+
+		private void InternalSetContent(UIElement content)
+		{
+			if (_window == null)
+			{
+				_rootBorder = new Border();
+				//_rootScrollViewer = new ScrollViewer()
+				//{
+				//	VerticalScrollBarVisibility = ScrollBarVisibility.Disabled,
+				//	HorizontalScrollBarVisibility = ScrollBarVisibility.Disabled,
+				//	VerticalScrollMode = ScrollMode.Disabled,
+				//	HorizontalScrollMode = ScrollMode.Disabled,
+				//	Content = _rootBorder
+				//};
+				//// _popupRoot = new PopupRoot();
+				//_window = new Grid()
+				//{
+				//	Visual = Compositor.CreateContainerVisual(),
+				//	Children =
+				//	{
+				//		_rootScrollViewer,
+				//		// , _popupRoot
+				//	}
+				//};
+
+				_window = new Grid {
+					IsLoaded = true,
+					Children = { _rootBorder }
+				};
+
+				Compositor.RootVisual = _window.Visual;
+			}
+
+			_rootBorder.Child = _content = content;
+		}
+
+		private UIElement InternalGetContent() => _content;
+
+		private static Window InternalGetCurrentWindow()
+		{
+			if (_current == null)
+			{
+				_current = new Window();
+			}
+
+			return _current;
+		}
+
+		internal IDisposable OpenPopup(Popup popup)
+		{
+			if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Debug))
+			{
+				this.Log().Debug($"Creating popup");
+			}
+			
+			var popupChild = popup.Child;
+			//_popupRoot.Children.Add(popupChild);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)